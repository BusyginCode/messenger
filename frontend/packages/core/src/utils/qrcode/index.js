import qrcode from 'qrcode-generator'

const make = input => {
  try {
    const qr = qrcode(0, 'M')
    qr.addData(input)
    qr.make()
    return qr
  } catch (e) {
    console.log(e)
  }
}

export default {
  qrcode,
  make
}
