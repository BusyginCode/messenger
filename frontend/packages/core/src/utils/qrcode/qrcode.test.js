import qrcode from './index'

describe('core/utils/qrcode', () => {
  it('should make correct', () => {
    expect(qrcode.make({})).toMatchSnapshot()
  })
})
