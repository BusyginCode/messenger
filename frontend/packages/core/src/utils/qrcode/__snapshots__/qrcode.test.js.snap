// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core/utils/qrcode should make correct 1`] = `
Object {
  "addData": [Function],
  "createASCII": [Function],
  "createDataURL": [Function],
  "createImgTag": [Function],
  "createSvgTag": [Function],
  "createTableTag": [Function],
  "getModuleCount": [Function],
  "isDark": [Function],
  "make": [Function],
  "renderTo2dContext": [Function],
}
`;
